### PR TITLE
Added page for redirection after login with open ID

### PR DESCRIPTION
**Pull request by petro97**
Description:

Linting:

- [x] No linting errors

Component Updates:

- [x] Component package requires update on npm [@elasticpath/store-components](https://www.npmjs.com/package/@elasticpath/store-components)

Tests:

- [x] E2E tests (npm test run with `e2e`)
- [x] Manual tests

Documentation:

- [x] Requires documentation updates

### CODE REVIEW

File: app/src/App.tsx [hunk 1]
```diff
@@ -130,11 +133,10 @@ const VersionContainer = (props) => {
   );
 };
 
-const Root = (props) => {
-  const { componentsData } = props;
+const Root = () => {
   const { error } = React.useContext(ErrorContext);
   return [
-    <VersionContainer key="version-container" componentsVersion={componentsData.version} appVersion={packageJson.version} />,
```
Comment (shaunmaharaj) [inline app/src/App.tsx]: why are we removing this?

File: app/src/App.tsx [hunk 2]
```diff
@@ -158,7 +160,7 @@ const Root = (props) => {
     <div key="app-content" className="app-content">
       <Switch>
         {routes.map(route => (
-          <RouteWithSubRoutes key={route.path} {...route} />
+          <RouteWithSubRoutes key={`${route.path}_${Math.random().toString(36).substr(2, 9)}`} {...route} />
```
Comment (shaunmaharaj) [inline app/src/App.tsx]: thanks for fixing!!

File: app/src/App.tsx (hunk 1 above)
Comment (shaunmaharaj) [reply app/src/App.tsx]: Ah, now I see why you removed this. I'll add a suggested snippet further below 😉

File: app/src/App.tsx [hunk 3]
```diff
@@ -174,7 +176,10 @@ const AppWithRouter = (props) => {
     <Router>
       <ErrorDisplayBoundary>
         <CountProvider>
-          <App componentsData={componentsData} />
+          <Switch>
+            <Route path="/loggedin" exact component={LoginRedirectPage} />
+            <Route path="/" exact={false} component={App} componentsData={componentsData} />
```
Comment (shaunmaharaj) [inline app/src/App.tsx]: This doesn't actually pass the componentsData prop to the component unfortunately. Router doesn't seem to pass them 😞 

Here's an alternative you can do which does using render:
```suggestion
            <Route path="/" exact={false} render={passed_props => <App {...passed_props} componentsData={componentsData} />} />
```

Comment (petro97) [reply app/src/App.tsx]: Thank you @shaunmaharaj

**Pull request merged**